### PR TITLE
Issues

### PR DESCRIPTION
**Pull request by upexcel**

### CODE REVIEW

File: src/pages/register/register.ts [hunk 1]
```diff
@@ -46,13 +46,13 @@ export class RegisterPage implements OnInit {
...
     }
     signin(logvalue: any) {
-        this.spin = true;
         this._login.getLogin(logvalue).then((res) => {
             this.data=res;
             if (this.data.status === 1) {
                 this.data = res;
                 this._appConfigService.setUserData(res.data);
-                this._navCtrl.setRoot(HomePage,{"access_token":this.data.data.access_token});
+                 this._toast.toast("Wellcome "+ logvalue.firstname,3000);
```
Comment (excellencetechnologies) [inline src/pages/register/register.ts]: Welcome

File: src/pages/register/register.ts [hunk 2]
```diff
@@ -46,13 +46,13 @@ export class RegisterPage implements OnInit {
...
+                 this._navCtrl.setRoot(HomePage,{"access_token":this.data.data.access_token});
```
Comment (excellencetechnologies) [inline src/pages/register/register.ts]: what is this? we should get access token from local storage?

File: src/pages/register/register.ts (hunk 1 above)
Comment (upexcel) [inline src/pages/register/register.ts]: fixed

File: src/pages/register/register.ts (hunk 2 above)
Comment (upexcel) [inline src/pages/register/register.ts]: fixed

**Pull request merged**